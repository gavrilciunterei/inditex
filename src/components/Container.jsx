import React from 'react';

function Container({ children }) {
  return <div className="container px-5 py-6 mx-auto">{children}</div>;
}

export default Container;
