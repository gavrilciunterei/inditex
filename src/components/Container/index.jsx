import React from 'react';

function Container({ children }) {
  return (
    <div className="container lg:px-24 sm:px-10 py-6 mx-auto flex flex-col">
      {children}
    </div>
  );
}

export default Container;
