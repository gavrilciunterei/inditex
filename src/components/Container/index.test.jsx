import React from 'react';
import renderer from 'react-test-renderer';
import Container from './index';

it('Test Container', () => {
  const tree = renderer
    .create(
      <Container>
        <div>
          <h1>Prueba container</h1>
        </div>
      </Container>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
