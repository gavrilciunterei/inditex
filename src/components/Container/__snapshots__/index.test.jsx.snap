// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Container 1`] = `
<div
  className="container lg:px-24 sm:px-10 py-6 mx-auto flex flex-col"
>
  <div>
    <h1>
      Prueba container
    </h1>
  </div>
</div>
`;
